CHANGELOG.md

```diff
@@ -1,109 +1,197 @@
-# 1.0.0-rc1
-
-## November 6, 2022
-
-1.0.0 is a breaking update to ACA-Py whose version is intended to indicate the
-maturity of the implementation. The final 1.0.0 release will be Aries Interop
-Profile 2.0-complete, and based on Python 3.7 or higher.
-
-### Breaking Changes
-
-As of release candidate 1.0.0-rc1, the only identified breaking change is the
-handling of "unrevealed attributes" during verification (see
-[\#1913](https://github.com/hyperledger/aries-cloudagent-python/pull/1913) for
-details). As few implementations of Aries Wallets support unrevealed attributes
-in an AnonCreds presentation, this is unlikely to impact any deployments.
-
-### Categorized List of Pull Requests
-
-In rc1, there are not a lot of new features, as the focus is on cleanup and
-optimization. The biggest is the inclusion with ACA-Py of a universal resolver
-interface, allowing an instance to have both local resolvers for some DID
-Methods and a call out to an external universal resolver for other DID Methods.
-Another significant feature is full support for Hyperledger Indy transaction
-endorsement for Authors and Endorsers. A new repo
+# 0.8.0-rc0
+
+## February 8, 2023
+
+0.8.0 is a breaking change that contains all updates since release 0.7.5. It
+extends the previously tagged `1.0.0-rc1` release because it is not clear when
+that release will be finalized. Many of the PRs in this release were previously
+included in the `1.0.0-rc1` release. The categorized list of PRs separates those
+that are new from those in the `1.0.0-rc1` release candidate.
+
+With this release, a new automated process publishes container images in the
+Hyperledger container image repository. New images for the release are
+automatically published by the GitHubAction Workflows: [publish.yml] and
+[publish-indy.yml]. The actions are triggered when a release is tagged, so no
+manual action is needed. The images are published in the [Hyperledger Package
+Repository under aries-cloudagent-python] and a link to the packages added to
+the repositories main page (under "Packages"). Additional information about the
+container image publication process can be found in the document [Container
+Images and Github Actions].
```

Comment:
```suggestion
container image publication process can be found in the document [Container Images and Github Actions](https://github.com/hyperledger/aries-cloudagent-python/blob/main/ContainerImagesAndGithubActions.md).
```

Reply:
That didn't format well.  I was trying to point out the missing link for `[Container Images and Github Actions]` and recommend it be changed to `[Container Images and Github Actions](https://github.com/hyperledger/aries-cloudagent-python/blob/main/ContainerImagesAndGithubActions.md)`

Reply:
Fixed — I missed copying over the link targets from the publishing doc.